variables.robot

```diff
@@ -462,6 +474,8 @@ ${OS_UBUNTU}=               ubuntu
 @{CONFIG45}=                &{RTE53}    &{USB11}    &{SSD15}
 @{CONFIG46}=                &{RTE54}    &{USB11}    &{SSD07}
 @{CONFIG47}=                &{RTE63}    &{EMMC01}
+@{CONFIG48}=                &{RTE64}    &{SSD16}
+@{CONFIG49}=                &{RTE65}
```

Comment:
No ssd in this one?

Reply:
Updated in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/424/commits/a5400a1df5c50c438d138565924e5ab2a6d9c75f)